src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/** This abstract class is a generic class which all other command classes inherit from. */
+public abstract class Command {
+    protected boolean exit;
+
+    /**
+     * @return Boolean representing whether the program should terminate after executing the command.
```

Comment:
Should there be a description for this method in the Javadocs? 🤔 Maybe you could also check for subsequent classes too (i.e. `Parser.java`)!